addons/uh60_weapons/config/cfgMagazines.hpp

```diff
@@ -45,7 +45,20 @@ class cfgMagazines {
     hardpoints[] =    {"VTX_ST_L"};
     model = \z\vtx\addons\uh60_weapons\Data\M230\M230_L.p3d;
   };
-  class VTX_M230_Chaingun_R: VTX_M230_Chaingun_L {
+  class VTX_M230_Chaingun_R: VehicleMagazine {
```

Comment:
why did this need to be explicitly redefined instead of inherited?

Reply:
There was an issue with duplicating the class for AP ammo.  Thinking about a potential solution to the problem now